security/certificate_types_descriptions/service-ca-certificates.adoc

```diff
@@ -27,6 +27,11 @@ The service CA expiration of 26 months is longer than the expected upgrade inter
 A manually-rotated service CA does not maintain trust with the previous service CA. You might experience a temporary service disruption until the pods in the cluster are restarted, which ensures that pods are using service serving certificates issued by the new service CA.
 ====
 
+[IMPORTANT]
+====
+Applications that use a different trust store, such as JavaKeyStore, need to be capable of accepting new CA certifications dynamically into it's trust store, or else when automated rotation occurs, pods will require a restart.
```

Comment:
> pods will require

"pods" reads unclear. How about explicitly "the application pods will require"?

LGTM except commented minor issues.

```
PTAL: @xingxingxia @ABU Kashem
```


@lpettyjo BTW, service-ca falls in Auth, Auth Dev lead is `@stlaz`, not Abu Kashem (he is Apiserver Dev lead instead). And even his GH ID is not "`@ABU Kashem`", he is `@tkashem` instead.

Reply:
@xingxingxia Thanks! I was given Abu as the contact. Thanks for the correction!